manifests/traefik/crds/serverstransports.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: serverstransports.traefik.containo.us
+spec:
+  group: traefik.containo.us
+  version: v1alpha1
+  names:
+    kind: ServersTransport
+    plural: serverstransports
+    singular: serverstransport
+  scope: Namespaced
```

Comment:
Yeah, I know. No newline at end of file. This file was automatically fetched with `make update-crds-traefik` which downloads the helm chart and extracts the CRDs. I do not modify these from upstream.